static/js/popovers.js

```diff
@@ -1353,7 +1168,6 @@ export function hide_all_except_sidebars(opts) {
     } else if (opts.exclude_tippy_instance) {
         hideAll({exclude: opts.exclude_tippy_instance});
     }
-    hide_actions_popover();
```

Comment:
Can we call this as `tippy.hideAll()`, or otherwise make it clear that this is hiding all Tippy tooltips and popovers? Might want a comment explaining that popover_menus.js is implemented using Tippy and thus this function hides those.